lib/LedgerSMB/Routes/ERP/API/Products.pm

```diff
@@ -555,7 +617,19 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/Warehouse'
+        400:
+          $ref: '#/components/responses/400'
+        401:
+          $ref: '#/components/responses/401'
+        403:
+          $ref: '#/components/responses/403'
+        404:
+          $ref: '#/components/responses/404'
     post:
+      tags:
+        - Warehouses
+      summary: Put a list of warehouses
```

Comment:
```suggestion
      summary: Create a warehouse
```

Reply:
We're not putting a single one but a list

Reply:
I'm not sure what you're saying, but usually a "POST" to the parent resource creates a single new child resource. That's what's happening here too. Just check the schema of the request body: `#/components/schemas/NewWarehouse` (which is a single object, not a list).

Reply:
Fixed